packages/wallet-utils/src/lib/wallet-utils.ts

```diff
@@ -0,0 +1,125 @@
+import { utils, transactions as nearTransactions, Signer } from "near-api-js";
+import {
+  Transaction,
+  Action,
+  ProviderService,
```

Comment:
To avoid exposing this internal service, we should consider using the JsonProvider from `near-api-js` directly